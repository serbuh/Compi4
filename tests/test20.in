// While statement with break

C x = 0 C;
print "Water is frozen";

while(x < 200 C) {
  x = x + 20 C;
  print "Warming up water...";
  if(x == 100 C) {
    print "Water is boiling, cannot warm up";
    break;
  }
}
print "Water boiling complete!";
