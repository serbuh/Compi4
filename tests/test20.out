Water is frozen
Warming up water...
Warming up water...
Warming up water...
Warming up water...
Warming up water...
Water is boiling, cannot warm up
Water boiling complete!
